sound/soc/intel/boards/sof_sdw.c

```diff
@@ -574,7 +574,6 @@ static struct sof_sdw_codec_info codec_info_list[] = {
 			},
 		},
 		.dai_num = 1,
-		.codec_type = SOF_SDW_CODEC_TYPE_JACK,
```

Comment:
@bardliao can you please give an example of what the dai_link names become before and after this change?

Reply:
> @bardliao can you please give an example of what the dai_link names become before and after this change?

For example, if a codec supports JACK on dai 0 and AMP on dai 1, the existing code will create dailinks
SDW0-Playback-SimpleJack or SDW0-Playback-SmartAmp for both dailinks,
and it will be SDW0-Playback-SimpleJack for dailink 0 and SDW0-Playback-SmartAmp for dailink 1 after this change.
I will add it to the commit message.